src/sentry/feedback/blueprints/api.md

```diff
@@ -0,0 +1,303 @@
+| feedback.contact_email     | string           | -                                     |
+| feedback.message           | string           | -                                     |
+| feedback.replay_id         | optional[string] | -                                     |
+| feedback.url               | string           | -                                     |
+| platform                   | string           | -                                     |
+| release                    | string           | -                                     |
+| request                    | optional[object] | -                                     |
+| request.headers            | optional[object] | -                                     |
+| request.headers.User-Agent | optional[string] | -                                     |
+| sdk                        | object           | -                                     |
+| sdk.name                   | string           | -                                     |
+| sdk.version                | string           | -                                     |
+| tags                       | optional[object] | -                                     |
+| timestamp                  | number           | UTC timestamp.                        |
+| user                       | optional[object] | -                                     |
+| user.email                 | optional[string] | -                                     |
+| user.id                    | optional[string] | -                                     |
+| user.ip_address            | optional[string] | -                                     |
+| user.name                  | optional[string] | -                                     |
+| user.username              | optional[string] | -                                     |
+
+- Request
+
+  ```json
+  {
+    "dist": "abc123",
+    "environment": "production",
+    "event_id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+    "feedback": {
+      "contact_email": "colton.allen@sentry.io",
+      "message": "I really like this user-feedback feature!",
+      "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+      "url": "https://docs.sentry.io/platforms/javascript/"
+    },
+    "platform": "javascript",
+    "release": "version@1.3",
+    "request": {
+      "headers": {
+        "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
+      }
+    },
+    "sdk": {
+      "name": "sentry.javascript.react",
+      "version": "6.18.1"
+    },
+    "tags": {
+      "key": "value"
+    },
+    "timestamp": 146990235.23,
+    "user": {
+      "email": "username@example.com",
+      "id": "123",
+      "ip_address": "127.0.0.1",
+      "name": "user",
+      "username": "user2270129"
+    }
+  }
+  ```
+
+## User Feedback [/projects/<organization_slug>/<project_slug>/user-feedback/<user_feedback_id>/]
```

Comment:
should the URL be `/feedback/` rather than `/user_feedback/` cc @JoshFerge

Reply:
yes!